src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Color;
+  color : Shape;
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color : Color,
+    public a:number,
+    public b:number,
+    public c:number,
```

Comment:
```suggestion
    public color: Color,
    public a: number,
    public b: number,
    public c: number,
```
Fix formatting everywhere